xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -82,10 +82,16 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="allowedRanges">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="allowedRanges">The set of characters that the encoder is allowed to not encode.</param>
+        <summary>Creates a new instance of the HtmlEncoder class specifying characters to encode.</summary>
+        <returns>A new instance of the <see cref="T:System.Text.Encodings.Web.HtmlEncoder"/> class.</returns>
+        <remarks><format type="text/markdown"><![CDATA[ 
+
+## Remarks  
+ Some characters in `allowedRanges` might still be encoded, that is, this parameter indicates what ranges the encoder is allowed to not encode, not what characters it must not encode.  
```

Comment:
encoded; that is, 
Again, this comment applies to all the encoder Create(UnicodeRange[]) methods.